src/tests/roc_sndio/target_sox/test_sox_source.cpp

```diff
@@ -214,15 +223,17 @@ TEST(sox_source, pause_restart) {
 
 TEST(sox_source, eof_restart) {
     core::TempFile file("test.wav");
-
+    const core::nanoseconds_t BufDuration =
+        FrameSize * core::Second / (SampleRate * packet::num_channels(ChMask));
```

Comment:
It seems this constant is not needed and we can just use FrameDuration instead?

Reply:
done.